config/images/images.yaml

```diff
@@ -66,6 +66,18 @@ images:
   tags:
   - v1.24.1
   - v1.26.4
+- source: ghcr.io/credativ/vali
+  destination: eu.gcr.io/gardener-project/3rd/credativ/vali
```

Comment:
Aa, I was about to miss the most important. GCR domain has to be switched with the AR.

PS: Sorry for not noticing earlier.

Reply:
I've updated the path.